app/src/main/java/com/amaze/filemanager/ui/activities/MainActivity.java

```diff
@@ -1857,7 +1857,7 @@ public void modify(String oldpath, String oldname, String newPath, String newnam
   @Override
   public void onPreExecute(String query) {
     final MainFragment mainFragment = getCurrentMainFragment();
-    Objects.requireNonNull(mainFragment);
+    if (mainFragment == null) return;
```

Comment:
I think before returning null, should show a toast saying 'Operation unsuccessful' or something similar.